src/content/app/species/state/general/speciesGeneralHelper.ts

```diff
@@ -199,6 +197,15 @@ type StatsFormattingOption = {
   secondaryUnit?: string;
   primaryValuePostfix?: string;
   helpText?: string;
+  getHelpText?: (allStats: SpeciesStatistics) => string;
+};
+
+const getComparaCoverageHelpText = (allStats: SpeciesStatistics) => {
+  const referenceSpeciesName = allStats.homology_stats.reference_species_name;
+  if (!referenceSpeciesName) {
+    return '';
+  }
+  return `Proportion of genes with orthologs with ${referenceSpeciesName}`;
```

Comment:
Shouldn't it be `orthologues` instead of `orthologs` here?

Reply:
This is the text from the ticket (see [ENSWBSITES-2288](https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2288)). I don't think we do -ues for orthologs, homologs, paralogs, etc.

Reply:
I only see `orthologues' in the [current site](https://plants.ensembl.org/Arabidopsis_thaliana/Gene/Compara_Ortholog/pan_compara?g=AT5G06870;r=5:2133918-2135168;t=AT5G06870.1#list_no_ortho) as well as the text used for the same tooltip in the [beta site](https://beta.ensembl.org/species/grch38)

Reply:
As discussed during the scrum today, let's leave it as it is for now.